Automattic/jetpack
# Social Menu & Icon: Add Ravelry Icon

Fixes an issue raised in p8HJWS-ui-p2

#### Changes proposed in this Pull Request:

* Adds support for Ravelry to the Social Menu and widget. According to the original discussion, it has surpassed 9 million users.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Adds to an existing feature

#### Testing instructions:






Go to the Customizer with a theme that supports social menus and add a social menu using the link `ravelry.com`, and do the same with the Social Icons widget, then verify the icon displays as expected. 

<img width="526" alt="Screenshot 2020-03-20 at 12 02 25" src="https://user-images.githubusercontent.com/43215253/77162827-bc075780-6aa4-11ea-93d7-d311bf32d4b1.png">

#### Proposed changelog entry for your changes:

* Social Menus: Add Ravelry support
